src/routes/(console)/onboarding/create-project/+page.svelte

```diff
@@ -31,6 +44,9 @@
                 teamId,
                 isCloud ? region : undefined
             );
+
+            await markOnboardingComplete();
```

Comment:
doe we need to await it?

Reply:
maybe, maybe not.
I don't want to really wait for an edge case to happen where this fails for some reason and the next hit to onboarding route creates another org, if the last was deleted. wdyt?

Reply:
but this should never make project creation fail though 👍🏻

Reply:
maybe move it to `finally`, while being non await?

Reply:
then it doesnt make a difference anyway if its awaited :-P just makes everything slower